docs/HowTo/virtualbox.md

```diff
@@ -0,0 +1,83 @@
+# VirtualBox
+How to set up your VirtualBox installation.
+
+## Introduction
+### Use cases
+Consider the following examples:
+- You use software that doesn't work on your EuroLinux 8 installation.
+- You want to try out some potentially dangerous actions and don't want to
+  endanger your machine.
+- You develop some awesome software and want to test it on several systems for
+  compatibility
+- You want an easily reproducible environment that works the same way on every
+  person's machine 
+- You need a multi-machine laboratory, maybe with several different systems and
+  don't have the resources for a physical equipment
+
+### Why use it over other virtualisation providers?
+VirtualBox is the provider with a copyleft license and a focus on
+interoperability when it comes to supporting different platforms. This allows
+you to cooperate with someone running a different operating system and when
+exchanging documentation - once written it's applicable to anyone that can run
+the software.  
+New VirtualBox versions usually support older operating systems too. One can
+enjoy the new features on a system, which has recently reached its End of Life,
+which can be indispensable for a company that can't migrate yet.
+
+## System requirements
+A brief documentation is available at
+[Upstream's](https://www.virtualbox.org/wiki/End-user_documentation).  
+
+Your machine shall support hardware virtualisation. If it doesn't, you either
+need to perform additional troubleshooting - e.g. enable virtualisation in your
+machine's BIOS settings.
+
+```
+[ "$(grep -c vmx /proc/cpuinfo)" -gt 0 ] && echo "OK"
+```
+
+Make sure you're running EuroLinux 8 on x86_64 architecture rather than ARM.  
+
+```
+[ "$(lscpu | grep x86_64)" ] && echo "OK"
+```
+
+## Installation on EuroLinux 8
+
+As of today, 6.1 is the main VirtualBox branch and this is the one we install
```

Comment:
today( 2 sep 2021 ), it just make it cleaner. 

You can also add link to release pages - https://www.virtualbox.org/wiki/Linux_Downloads and note that you should use version for RHEL8

Reply:
Release page added to "Additional resources" section along with new information on installation from RPM Fusion.